teamscale_client/data.py

```diff
@@ -61,11 +61,12 @@ def __init__(self, findings, path, content=None):
         self.content = content
 
 
-FindingDescription = collections.namedtuple('FindingDescription', ['typeid', 'description', 'enablement'])
+FindingDescription = collections.namedtuple('FindingDescription', ['typeid', 'name', 'description', 'enablement'])
```

Comment:
I think we should move the name parameter to the end and make sure it can be `None` automatically, because old clients will otherwise get a completely broken findings description